build.gradle

```diff
@@ -995,7 +1000,7 @@ task jar2(type: Sync) {
     description = 'Create minimal executable output for debugging.'
     destinationDir = file("${buildDir}/install/jar2")
     from jar
-    from(configurations.runtime) {
+    from(configurations.runtimeOnly) {
```

Comment:
I will simply remove `jar2`. It is a remnant of something we had when we were using Ant. Didier requested we have this because it's supposedly faster, but I find that `./gradlew installDist` is plenty fast.